common/middleware/set-current-location.js

```diff
@@ -0,0 +1,16 @@
+const { get } = require('lodash')
+
+module.exports = function currentLocation (req, res, next) {
+  if (req.session.currentLocation) {
+    return next()
+  }
+
+  const userLocations = get(req.session, 'user.locations', [])
+
+  if (userLocations.length === 1) {
```

Comment:
Should we just default to the first one in this case? Otherwise the service will be broken until the user can choose.

Reply:
Not a bad shout